tests/pathfinding/test_capacity_updates.py

```diff
@@ -0,0 +1,506 @@
+"""
+The tests in this module mock events creation by using a mock blockchain listener.
+
+The Capacity Updates should be tested in various states
+"""
+
+from typing import List
+
+import pytest
+from eth_utils import decode_hex
+
+from pathfinding_service import PathfindingService
+from pathfinding_service.exceptions import MessageException
+from pathfinding_service.model import TokenNetwork
+from raiden.messages import UpdatePFS
+from raiden.utils import CanonicalIdentifier
+from raiden.utils.typing import ChainID, ChannelID, Nonce, TokenAmount, TokenNetworkAddress
+from raiden_libs.types import Address, ChannelIdentifier
+
+
+DEFAULT_TOKEN_NETWORK_ADDRESS = Address('0x6e46B62a245D9EE7758B8DdCCDD1B85fF56B9Bc9')
+DEFAULT_TOKEN_NETWORK_ADDRESS_BYTES = TokenNetworkAddress(
+    decode_hex(DEFAULT_TOKEN_NETWORK_ADDRESS),
+)
+DEFAULT_TOKEN_ADDRESS = Address('0x44Ac22fd9672cC559Ab171603D474cEA8a2D7b4D')
+
+"""
+def get_UpdatePFS_message(
+        canonical_identifier: int = DEFAULT_CANONICAL_IDENTIFIER,
+        updating_participant,
+        other_participant,
+        updating_nonce: typing.Nonce,
+        other_nonce: typing.Nonce,
+        updating_capacity: typing.TokenAmount,
+        other_capacity: typing.TokenAmount,
+        reveal_timeout: int,
+        signature: typing.Optional[typing.Signature] = None,
+) -> UpdatePFS:
+    capacity_update =
+
+    def get_signed_monitor_request(
+            nonce: int = 5,
+            reward_amount: int = DEFAULT_REWARD_AMOUNT,
+            closing_privkey: str = DEFAULT_PRIVATE_KEY1,
+            nonclosing_privkey: str = DEFAULT_PRIVATE_KEY2,
+    ) -> MonitorRequest:
+        bp = HashedBalanceProof(  # type: ignore
+            channel_identifier=DEFAULT_CHANNEL_IDENTIFIER,
+            token_network_address=DEFAULT_TOKEN_NETWORK_ADDRESS,
+            chain_id=1,
+            balance_hash='',
+            nonce=nonce,
+            additional_hash='',
+            priv_key=closing_privkey,
+        )
+        monitor_request = UnsignedMonitorRequest.from_balance_proof(
+            bp,
+            reward_amount=reward_amount,
+        ).sign(nonclosing_privkey)
+        return monitor_request
+"""
+
+
+def test_pfs_rejects_capacity_update_with_wrong_chain_id(
+    addresses: List[Address],
+    pathfinding_service_mocked_listeners: PathfindingService,
+):
+
+    message1 = UpdatePFS(
```

Comment:
why not call it `message`? Same for all tests below.

Reply:
done